.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
```

Comment:
**suggestion:** the ".idea" folder is also usually ignored since its specific for the developer IDE